lib/app_web/live/imgup_live.ex

```diff
@@ -0,0 +1,85 @@
+defmodule AppWeb.ImgupLive do
+  use AppWeb, :live_view
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(:uploaded_files, [])
+     |> allow_upload(:image_list, accept: ~w(image/*), max_entries: 6, chunk_size: 64_000, max_file_size: 5_000_000, external: &presign_upload/2)}
+  end
+
+  # Adding presign for each entry for S3 upload --------
+
+  defp presign_upload(entry, socket) do
+    uploads = socket.assigns.uploads
+    bucket_original = "imgup-original"
+    bucket_compressed = "imgup-compressed"
+    key = Cid.cid("#{DateTime.utc_now() |> DateTime.to_iso8601()}_#{entry.client_name}")
```

Comment:
the `key` should be the `cid` (hash) of the actual content, not a `DateTime`. 
please see: https://github.com/dwyl/imgup/issues/62

Using a `DateTime` means the filename will always be unique. We don't want that. 
We want to _avoid_ duplicates of files being uploaded.